spec/DemographicData.csvschema.json

```diff
@@ -0,0 +1,9 @@
+{
+    "columns": {
+        "country": "String",
+        "age": "String",
+        "year": "Int",
+        "gender": "String",
+        "value": "Decimal"
+    }
+}
```

Comment:
This age range field modeled on the format of the unwpp data, and both bounds are inclusive (resulting in the strange looking values 0-0, 1-1, 2-2 when the age step is 1!) But that's not to say it's the best way, might be worth running it by Wes again.